cli/cmd/develop.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/omni-network/omni/lib/errors"
+
+	"github.com/spf13/cobra"
+)
+
+func newDevelopCmds() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "develop",
+		Short: "XApp development commands",
+		Args:  cobra.NoArgs,
+	}
+
+	cmd.AddCommand(
+		newForgeProjectCmd(),
+	)
+
+	return cmd
+}
+
+func newForgeProjectCmd() *cobra.Command {
+	var projName string
+
+	cmd := &cobra.Command{
+		Use:   "new",
+		Short: "Scaffold a Forge project",
+		Long: `Scaffold a new Forge project with a XGreeter contract
+accompanied by simple mocked testing and a multi-chain deployment script.`,
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return newProjectTemplate(projName)
+		},
+	}
+
+	bindNewProjConfig(cmd, &projName)
+
+	return cmd
+}
+
+// newProjectTemplate creates a new project using the forge template.
+func newProjectTemplate(projectName string) error {
+	// Check if forge is installed
+	if !checkForgeInstalled() {
+		// Forge is not installed, return an error with a suggestion.
+		return &cliError{
+			Msg:     "forge is not installed.",
+			Suggest: "You can install foundry by visiting https://github.com/foundry-rs/foundry",
+		}
+	}
+
+	// Execute the `forge init` command.
+	cmd := exec.Command("forge", "init", "--template", "https://github.com/omni-network/omni-forge-template.git")
```

Comment:
Can this just be part of `go generate ./...`?

Reply:
Is this going to be used by our operators?

Reply:
Could you explain further?

Reply:
What I meant is:If this is about using forge tools next to our Go codebase, then we can set this up as `go generate` command.

Reply:
I see! It's like 2) - this command is intended for developers that will write programs that will be run on the Omni EVM or be deployed to any of the supported rollups too